### PR TITLE
Add support to `useInput` for Page Up and Page Down keys

### PR DESCRIPTION
**Pull request by technicalpickles**
I'm working on a pager, and wanted to be able to check for page up / page down events, and thought it'd be a common enough thing to make a PR.

### CODE REVIEW

**Comment by vadimdemedes** [comment]
Thanks! Could you document it in readme too?

File: src/hooks/use-input.ts [hunk 1]
```diff
@@ -25,6 +25,16 @@ export interface Key {
 	 */
 	rightArrow: boolean;
 
+	/**
+	 * Page Down key was pressed
```
Comment (vadimdemedes) [inline src/hooks/use-input.ts:29]:
```suggestion
	 * Page Down key was pressed.
```

File: src/hooks/use-input.ts [hunk 2]
```diff
@@ -25,6 +25,16 @@ export interface Key {
...
+	 */
+	pageDown: boolean;
+
+	/**
+	 * Page Up key was pressed
```
Comment (vadimdemedes) [inline src/hooks/use-input.ts:34]:
```suggestion
	 * Page Up key was pressed.
```

**Comment by technicalpickles** [comment]
@vadimdemedes done!

**Comment by vadimdemedes** [comment]
Thanks!

**Pull request merged**